src/pages/patientView/clinicalInformation/lib/clinicalAttributesStyleConsts.js

```diff
@@ -0,0 +1,183 @@
+    const status = key in config ? config[key].primary : true;
+    const formatter =
+        key in config && 'formatter' in config[key]
+            ? config[key].formatter
+            : t => {
+                  return `${t}`;
+              };
+    const prefix = status !== true ? '' : ', ';
+
+    builder.push(prefix);
+    builder.push(formatter(value));
+    return builder.join('');
+};
+
+/**
+ *  Gets a color for the given key, value, and keys. 
+    If there a specified color for the key and value, return it.
+    If there is no such value, then get the color only specified for the key. 
+    Otherwise, return a default color. 
+ * @param {string} value 
+ * @param {string} key 
+ * @param {object} keys 
+ */
+export const colorGetter = (value, key, keys) => {
+    // Style based on attr-id only and not attr-value
+    const color = 'black';
+    const NO_VALUE = '_';
+
+    return key in keys
+        ? value in keys[key]
+            ? keys[key][value]
+            : NO_VALUE in keys[key]
+            ? keys[key][NO_VALUE]
+            : color
+        : color;
+};
+
+// Placeholder for configured/imported attributes
+export const config = {
+    OS_MONTHS: {
+        primary: false,
+        formatter: t => {
+            return `\xa0(${t} months)`;
+        },
+    },
+    DFS_MONTHS: {
+        primary: false,
+        formatter: t => {
+            return `\xa0(${t} months)`;
+        },
+    },
+    CANCER_TYPE_DETAILED: {
+        primary: false,
+        formatter: t => {
+            return `\xa0(${t})`;
+        },
+    },
+    PATIENT_DISPLAY_NAME: {
+        primary: false,
+        formatter: t => {
+            return `\xa0(${t})`;
+        },
+    },
+    DERIVED_SAMPLE_LOCATION: {
+        primary: false,
+        formatter: t => {
+            return `\xa0(${t})`;
+        },
+    },
+    SAMPLE_DISPLAY_NAME: {
+        primary: false,
+        formatter: t => {
+            return `\xa0(${t})`;
+        },
+    },
+    SERUM_PSA: {
+        primary: true,
+        formatter: t => {
+            return `, Serum PSA: ${t}`;
+        },
+    },
+    ERG_FUSION_ACGH: {
+        primary: true,
+        formatter: t => {
+            return `, ERG-fusion aCGH: ${t}`;
+        },
+    },
+    TMPRSS2_ERG_FUSION_STATUS: {
+        primary: true,
+        formatter: t => {
+            return `, TMPRSS2-ERG Fusion: ${t}`;
+        },
+    },
+    GLEASON: {
+        primary: true,
+        formatter: t => {
+            return `, Gleason: ${t}`;
+        },
+    },
+    AGE: {
+        primary: true,
+        formatter: t => {
+            return `${t} years old`;
+        },
+    },
+};
+
+// Colors taken from clinicalAttributes.scss
+export const colors = {
+    sample_color_primary: 'black',
+    sample_color_recurrence: 'orange',
+    sample_color_metastasis: 'red',
+    sample_color_cfdna: 'blue',
+    sample_color_xenograft: 'pink',
+};
+
+// Attributes with special colors
+export const keys = {
+    OS_STATUS: {
```

Comment:
how come these keys are not included in the above config object?

Reply:
good question: I did not think of it initially but I shall refactor that!